novawallet/Common/Network/Subquery/SubqueryHistoryOperationFactory.swift

```diff
@@ -65,7 +65,13 @@ final class SubqueryHistoryOperationFactory {
         }
 
         if filter.contains(.rewardsAndSlashes) {
-            filterStrings.append("{ reward: { isNull: false } }")
+            let filter = SubqueryInnerFilter(inner:
+                SubqueryCompoundFilter.or([
+                    SubqueryIsNotNullFilter(fieldName: "reward"),
+                    SubqueryIsNotNullFilter(fieldName: "poolReward")
```

Comment:
This will cause request to fail on networks without nominations pools. We faced the same bug on Andrid